doc/plugins/lineprofiler.rst

```diff
@@ -0,0 +1,53 @@
+####################
+Spyder Line Profiler
+####################
+
+**Spyder-Line-Profiler** is a plugin to run the Python `line_profiler <https://pypi.python.org/pypi/line_profiler>`_.
+
+.. image:: /images/lineprofiler/lineprofiler-overview.png
+   :alt: Spyder with the Line Profiler pane open
+
+Using the terminal inside Spyder allows you to use the system shell rather than just the IPython console. You can use it to issue commands, interact
+with version control or to run programs.
+
+============================
+Installing the Line Profiler
+============================
+
+If you installed Spyder using conda, the best way to install Spyder-terminal is to run the following command in your terminal or Anaconda prompt on Windows:
+
+.. code-block:: bash
+
+   conda install spyder-line-profiler -c spyder-ide
+
+.. important::
+
+   At the moment it is not possible to use this plugin with the Spyder :ref:`standalone_installers_ref` for Windows and macOS. We're working to make that possible in the future.
+
+Restart Spyder in order to be able to use the plugin.
+
+=======================
+Using the Line Profiler
+=======================
+
+When the Line Profiler is installed, it will be available under the menu item :menuselection:`View --> Panes --> Line Profiler`.
+
+.. image:: /images/lineprofiler/lineprofiler-view-panes.png
+   :alt: Spyder showing View Panes Line Profiler
+
+You will see it then as a tab next to the 'Files' tab. You can then either select a script with the button present in the pane or run the profiler from :menuselection:`Run --> Profile line by line`.
+your script line by line.
+
+.. image:: /images/lineprofiler/lineprofiler-run-menu.png
+   :alt: Spyder showing the line profiler button in the Run menu
+
+For the Line Profiler to work, we must place the ``@profile`` decorator on top of the functions that we wish to profile.
+
+.. image:: /images/lineprofiler/lineprofiler-add-decorators.gif
+   :alt: Adding 'profile' decorators to script to profile it line by line.
+
+Now that the decorators have been added, we can run the Line Profiler from the button in the Pane or from the :menuselection:`Run` menu.
```

Comment:
Maybe add something here about the path of the profiler saying that the path displayed there is the path of the file that we are profiling?  @ccordoba12 what do you think?

Reply:
I'll wait for feedback before updating this section.

Reply:
Good suggestion @juanis2112! I agree with it.

Reply:
Fixed by b698589.

Reply:
Looks like this was resolved, thanks.